doc-content/enterprise-only/vehicle-route-planning/run-vrp-manually-proc.adoc

```diff
@@ -49,26 +50,26 @@ $HOME/{data-dir-name}
     └── belgium-latest.osm.pbf
 ----
 // TODO maybe replace this with a screenshot, doesn't look good in PDF.
+. Change directory to `{PRODUCT_INIT}-{PRODUCT_VERSION_LONG}-optaweb-vehicle-routing/sources/optaweb-vehicle-routing-standalone/target`.
 . To run OptaWeb Vehicle Routing, enter the following command:
 +
 [source,subs="attributes+"]
 ----
-java -jar optaweb-vehicle-routing-standalone-{PRODUCT_VERSION}.jar \
---app.persistence.h2-dir=$HOME/<DATA_DIR_NAME>/db \
---app.routing.gh-dir=$HOME/<DATA_DIR_NAME>/graphhopper
---app.routing.osm-dir=$HOME/<DATA_DIR_NAME>/openstreetmap \
---app.routing.osm-file=<COUNTRY>-latest.osm.pbf \
---app.region.country-codes=<REGION_CODE>
+java -jar optaweb-vehicle-routing-standalone-{MAVEN_ARTIFACT_VERSION}.jar \
+--app.persistence.h2-dir=$HOME/{VRP-DATA-DIR}/db \
+--app.routing.gh-dir=$HOME/{VRP-DATA-DIR}/graphhopper \
+--app.routing.osm-dir=$HOME/{VRP-DATA-DIR}/openstreetmap \
+--app.routing.osm-file=<OSM_FILE_NAME>-latest.osm.pbf \
```

Comment:
The full file name is moldova-latest.osm.pbf
Can we left <OSM_FILE_NAME> without anny latest.osm.pbf part?

Reply:
Updated